dll/win32/shell32/brfolder.cpp

```diff
@@ -408,8 +408,8 @@ BrFolder_Expand(
     ULONG ulFetched;
     while (S_OK == pEnum->Next(1, &pidlTemp, &ulFetched))
     {
-        if (!BrFolder_InsertItem(info, lpsf, pidlTemp, pidlFull, hParent))
-            break;
+        /* We need to ignore the return value of BrFolder_InsertItem to process next folders */
```

Comment:
```suggestion
        /* Ignore return value of BrFolder_InsertItem to avoid incomplete folder listing */
```
A bit better wording.